.github/workflows/package.yml

```diff
@@ -50,8 +45,27 @@ jobs:
 
       - name: Install dependencies
         if: ${{ matrix.os == 'windows' }}
+        env:
+          NPCAP_OEM_URL: ${{ secrets.NPCAP_OEM_URL }}
         shell: powershell
-        run: .\resources\packaging\windows\scripts\npcap.ps1 -ARCH ${{ matrix.arch }} -OEM "${{ secrets.NPCAP_OEM_URL }}"
+        run: |
+          Write-Host "::group::Npcap SDK"
```

Comment:
if Write-Host was used for debugging purposes it can now be removed, also considering the presence of the secret.

Reply:
I saw it's displayed as '***' but if ilogging is not needed anymore, we could remove all the Write-Host

Reply:
> if Write-Host was used for debugging purposes it can now be removed, also considering the presence of the secret.

The `Write-Host` PowerShell command is the same as the `echo` command in Unix-like systems. Wrapping the output in `::group` and `::endgroup` makes that section of GitHub logs grouped and hence foldable, so it makes it easier to review the action runs. You can check it out on my last Action "Install dependencies" section of any of the Windows builds.

> I saw it's displayed as '***' but if ilogging is not needed anymore, we could remove all the Write-Host

It's automatically displayed hidden by GitHub since it's a secret variable and shouldn't be revealed. However, I've modified this section to follow GitHub's [instructions](https://docs.github.com/en/actions/security-guides/encrypted-secrets) on using encrypted secrets.

Reply:
Got it, we can keep them then 👍